### ISSUE TITLE
Web Application for the project

### ISSUE THREAD

**Issue by aarpit1010** [aarpit1010/Real-Time-Face-Mask-Detector#7]
Hosting of this Model using flask or any other framework that you are comfortable in to a WEB Application so that you can access your real time model anywhere unlike your local system.
This should be a basic application which opens up the web-cam and detects masks.

Here are a few links for reference:
- https://towardsdatascience.com/how-to-deploy-a-pre-trained-keras-model-with-opencv-and-flask-86c9dab76a9c
- https://github.com/abhinavmaharana/Face-Mask-Detection/tree/webapp/Website

Multiple PRs will be accepted.
Also, attach the live link for the website
